README.md

```diff
@@ -362,6 +362,20 @@ Make sure lockfile version is higher than v1 (Node.js 15 or higher).
 
 See [docs/npm.md](docs/npm.md) for more details.
 
+### yarn
+
+<https://v3.yarnpkg.com/>
+
+Current version: v3
+
+Cachi2 supports Yarn by parsing [package.json](https://v3.yarnpkg.com/configuration/manifest/)
```

Comment:
We do parse the `package.json` file, but only to identify the yarn version. I think the text in the gomod block is more accurate: https://github.com/eskultety/cachi2/blob/yarn-docs/README.md?plain=1#L313-L315 

I don't really like that it says "we parse the file" and in the next sentence it says "we don't realy parse it", but I think it's best to acknowledge that we use the `yarn` command to get the list of dependencies.

Reply:
Reworded. How about now?